ssc/cmod_pvwattsv5.cpp

```diff
@@ -36,27 +35,28 @@ OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 enum pvwatts_tracking_input { FIXED_OPEN_RACK, FIXED_ROOF_MOUNT, ONE_AXIS_SELF_SHADED, ONE_AXIS_BACKTRACKED, TWO_AXIS, AZIMUTH_AXIS };
 
 static var_info _cm_vtab_pvwattsv5_part1[] = {
+
         /*   VARTYPE           DATATYPE          NAME                         LABEL                                               UNITS        META                      GROUP          REQUIRED_IF                 CONSTRAINTS                      UI_HINTS*/
         { SSC_INOUT,        SSC_NUMBER,      "system_use_lifetime_output",     "Run lifetime simulation",                    "0/1",        "",                       "Lifetime",            "?=0",                        "",                              "" },
         { SSC_INPUT,        SSC_NUMBER,      "analysis_period",                "Analysis period",                            "years",      "",                       "Lifetime",            "system_use_lifetime_output=1", "",                          "" },
         { SSC_INPUT,        SSC_ARRAY,       "dc_degradation",                 "Annual AC degradation",                      "%/year",    "",                        "Lifetime",            "system_use_lifetime_output=1", "",                          "" },
```

Comment:
I think it's confusing for this to be called dc_degradation but signify AC degradation

Reply:
I agree, but we can't change it in v5 for compatibility issues. I was trying to avoid renaming variables in v7 for ease of user transition from pvwattsv5 to pvwattsv7. The behavior of the variable in v7 is consistent with pvwattsv5. @cpaulgilman , @n8blair , thoughts on whether it would be better to rename the variable for clarity in v7, or keep the variable name for ease of upgrade?

Reply:
Actually, my bad. This has been added to v5 since our last release, so we can rename it however we want. I'll address it.

Reply:
I've left the name the same but changed the input to actually represent dc degradation in both pvwattsv5 and pvwattsv7. At the moment this is only available in the SDK or the fuel cell model- but this does open up the possibility of allowing lifetime mode in PVWatts if we choose to.